lib/configuration.yaml

```diff
@@ -314,6 +318,11 @@ mainnet_base: &mainnet_base
 
     dbSerializeVersion: 0
 
+  txValidationRules:
+    attribResrictEpoch: 2
+    addrAttribSize: 128
+    txAttribSize: 128
+
```

Comment:
@Jimbo4350, we'd need the same for the `mainnet_dryrun` (staging) and `testnet`

Reply:
Will do!